render-only-sample/rosumd/RosUmdResource.cpp

```diff
@@ -298,6 +298,13 @@ RosUmdResource::CalculateMemoryLayout(
                 m_hwFormat = RosHwFormat::X8888;
             }
 
+            // Force tiled layout for given configuration only
+            if ((m_usage == D3D10_DDI_USAGE_DEFAULT) &&
+                (m_bindFlags == D3D10_DDI_BIND_SHADER_RESOURCE))
+            {
+                m_hwLayout = RosHwLayout::Tiled;
+            }
+
             // Using system memory linear MipMap as example
             m_hwWidthPixels = m_mip0Info.TexelWidth;
             m_hwHeightPixels = m_mip0Info.TexelHeight;
```

Comment:
Hi, Marek:

   Please feel free to split the "alignment" calculation for linear and tiled resources.

                                            Thanks, Indy, 2/9/2016

            m_hwWidthTilePixels = VC4_BINNING_TILE_PIXELS;
            m_hwHeightTilePixels = VC4_BINNING_TILE_PIXELS;
            m_hwWidthTiles = (m_hwWidthPixels + m_hwWidthTilePixels - 1) / m_hwWidthTilePixels;
            m_hwHeightTiles = (m_hwHeightPixels + m_hwHeightTilePixels - 1) / m_hwHeightTilePixels;
            m_hwWidthPixels = m_hwWidthTiles*m_hwWidthTilePixels;
            m_hwHeightPixels = m_hwHeightTiles*m_hwHeightTilePixels;